### ISSUE TITLE
Update the rule help for the "combobox_autocomplete"

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1132]
Related to #1095 

This rule is to check if the popup of a Combobox contains an "aria-autocomplete" attribute. However, the current help is towards the "aria-autocomplete" of the combobox, rather than its popup. The "What to do" should be removing the aria-autocoplete attribute, rather than changing its value, from the popup, and also making sure the aria-automcomplete is set in the combobox element with a correct value. 

Branch: dev-autocomplete
PR: #1129

**Comment by philljenkins** [comment]
No box note needed for comments

**Comment by philljenkins** [comment]
The "What to do" should be removing the aria-autocoplete attribute, rather than changing its value, from the popup, and also making sure the aria-automcomplete is set in the combobox element with a correct value.
Current Help: [combobox_autocomplete.html](https://github.com/IBMa/equal-access/blob/dev-autocomplete/accessibility-checker-engine/help-v4/en-US/combobox_autocomplete.html), with changes noted:
### What to do
* If the combobox has autocomplete behavior that includes both inline text appearing after the user's cursor and a popup element, use `aria-autocomplete ="both"` on the text input element;
* **Or**, if the combobox has autocomplete behavior provided only through a popup element, ~~use `aria-autocomplete="list"`~~ remove the `aria-autocomplete` attribute from the pop-up;
* **Or**, if the combobox does not have autocomplete behavior, remove the `aria-autocomplete` attribute (the default value is `"none"`).

Also,
* Remove the `aria-autocomplete` attribute from popup elements of the combobox
* **And**, if autocomplete behavior is desired, verify that the text input element of the combobox has the `aria-autocomplete` attribute.

For example: 
```
<label for="comboInput">State</label>
<div class="combobox">
    <div class="cbPrimary">
    <input type="text" id="comboInput" class="cbEdit"
        role="combobox" aria-autocomplete="list"
        aria-haspopup="listbox" aria-expanded="true"
        aria-controls="popup_listbox" aria-activedescendant="cb-ak" />
    <button id="cbButton" tabindex="-1" aria-label="Open">▼</button>
    </div>
    <ul id="popup_listbox" role="listbox" aria-label="States">
    <li id="cb-al" role="option">Alabama</li>
    <li id="cb-ak" role="option" aria-selected="true">Alaska</li>
    </ul>
</div>
```

**Comment by philljenkins** [comment]
Rule ID [combobox_autocomplete.ts](https://github.com/IBMa/equal-access/blob/dev-autocomplete/accessibility-checker-engine/src/v4/rules/combobox_autocomplete.ts) messages:
- "group": "A combobox that supports autocompletion behavior must have the 'aria-autocomplete' attribute only on its text input element with a valid value; a value of '\"inline\"' is not supported"
- "Fail_1": "The combobox has the 'aria-autocomplete' attribute incorrectly set on an element within the popup referenced by \"{0}\"",
- "Fail_inline": "The combobox does not support an 'aria-autocomplete' attribute value set to '\"inline\"' ",
- "Pass": "The combobox does not use 'aria-autocomplete' value '\"inline\"' nor does it have 'aria-autocomplete' defined within the popup",

**Comment by philljenkins** [comment]
I'll include a link to the ARIA spec to the Help too https://w3c.github.io/aria/#aria-autocomplete

**Comment by philljenkins** [comment]
@shunguoy dev-autocomplete branch is ready for build and review of help changes

**Comment by shunguoy** [comment]
PR #1129 for @tombrunet  review

**Comment by philljenkins** [comment]
Spelling and grammar checker
![image](https://user-images.githubusercontent.com/16193609/207734224-b211427d-f08f-4acf-917c-1e32fdc6142a.png)
